src/Model.js

```diff
@@ -184,6 +183,59 @@ const Model = class Model {
     }
 
     /**
+     * Update many-many relations for model.
+     * @param relations
+     */
+    _refreshMany2Many(relations) {
+        const ThisModel = this.getClass();
+        const This = this;
```

Comment:
Ah sorry, i guess we misunderstood. What i meant:
_refreshMany2Many is already a class method, so since we use arrow functions for forEach and and filter we don't need the additional ```const This = this;``` so we can use this[name] etc. out of the box.